en/components/grids_templates/column_selection.md

```diff
@@ -0,0 +1,166 @@
+
+<div class="sample-container loading" style="height:570px">
+    <iframe id="grid-column-selection" data-src='{environment:demosBaseUrl}/grid/grid-column-selection' width="100%" height="100%" seamless frameborder="0" class="lazyload no-theming"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-column-selection-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+### Basic usage
+
+#### Interactions
+All of the presented columns are [`selectable`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selectable) by default. With that being said, in order to select a column, we just need to click over the targeted one to mark it as [`selected`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selected).
+
+> [!NOTE]
+> [`Multi Column Headers`](multi_column_headers.md) don't reflect on the [`selectable`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selectable) input. The [`IgxColumnGroupComponent`]({environment:angularApiUrl}/classes/igxcolumngroupcomponent.html) is [`selectable`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selectable), if at least one of it's children has that behaviour enabled. In addition, the component is marked as [`selected`]({environment:angularApiUrl}/classes/igxcolumngroupcomponent.html#selected) if all it's descendents are [`selected`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selected)
+
+<div class="sample-container loading" style="height:570px">
+    <iframe id="grid-column-group-selection" data-src='{environment:demosBaseUrl}/grid/grid-column-group-selection' width="100%" height="100%" seamless frameborder="0" class="lazyload no-theming"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-column-group-selection-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+#### Keyboard combinations
+Basically there are three scenarios for the keyboard navigation of the **Column Selection**:
+- Single selection - <kbd>click</kbd> over the column cell if it is **selectable**.
+- Multi column selection - holding <kbd>ctrl</kbd> + <kbd>click</kbd> over every **selectable** header cell.
+- Range column selection - holding <kbd>shift</kbd> + <kbd>click</kbd> selects all **selectable** columns in between.
+
+#### API manipulations
+The **API** provides some additional capabilities when it comes to the **non-visible** columns such that, every **hidden** column could be marked as [`selected`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selected) by setting the corresponding **setter**.
```

Comment:
such as marking every **hidden** column as [`selected`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selected) by setting the corresponding **setter**.